lib/measured/conversion.rb

```diff
@@ -50,6 +50,10 @@ def conversion_table
     @conversion_table ||= Measured::ConversionTable.new(@units).to_h
   end
 
+  def significant_digits
+    6
```

Comment:
This was chosen arbitrarily for now. Extracted to a method so it can be configurable.